lib/galaxy/model/store/load_objects.py

```diff
@@ -0,0 +1,76 @@
+import argparse
+import base64
+import logging
+import os
+import sys
+from typing import (
+    Any,
+    Dict,
+)
+
+import requests
+import yaml
+
+DESCRIPTION = """Load a Galaxy model store into a running Galaxy instance.
+
+See the corresponding galaxy-build-objects script for one possible way to
+create a model store to use with this script.
+
+This script "uploads" all datasets in a "discarded" state. To actually
+load the datasets into a Galaxy instance - the underlying libraries need
+to be used directly and fed your Galaxy's database configuration and
+objectstore setup.
```

Comment:
```suggestion
This script creates all datasets in a "discarded" state. To actually
load the datasets into a Galaxy instance's object store the underlying libraries need
to be used directly and fed your Galaxy's database configuration and
objectstore setup.
```
What libraries are you referring to ?